crates/extensions/c8y_mapper_ext/src/log_upload.rs

```diff
@@ -131,41 +140,36 @@ impl CumulocityConverter {
                 let event_response_id = self.http_proxy.send_event(create_event).await?;
 
                 // Send a request to the Uploader to upload the file asynchronously.
-                let uploaded_file_path = self
-                    .config
-                    .data_dir
-                    .file_transfer_dir()
-                    .join(target.external_id.as_ref())
-                    .join("log_upload")
-                    .join(format!("{}-{}", response.log_type, cmd_id));
-
-                let binary_upload_event_url = self
-                    .c8y_endpoint
-                    .get_url_for_event_binary_upload_unchecked(&event_response_id);
-
-                let upload_request = UploadRequest::new(
-                    self.auth_proxy
-                        .proxy_url(binary_upload_event_url.clone())
-                        .as_str(),
-                    &uploaded_file_path,
-                )
-                .with_content_type(ContentType::TextPlain);
+                let log_filename = format!("{}-{}", response.log_type, cmd_id);
 
-                self.uploader_sender
-                    .send((cmd_id.into(), upload_request))
-                    .await
-                    .map_err(CumulocityMapperError::ChannelError)?;
+                let uploaded_file_url = format!(
```

Comment:
Minor. The same comment as config_snapshot.
```suggestion
                let tedge_file_url = format!(
```